src/views/Home.jsx

```diff
@@ -14,11 +22,48 @@ export function Home({ createNewToken, setListToken }) {
 		navigate('/list');
 	};
 
+	const handleTokenChange = (e) => {
+		setExistingToken(e.target.value);
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const tokenExists = await getExistingList(existingToken);
+			if (tokenExists) {
+				setListToken(existingToken);
+				navigate('/list');
+			} else {
+				setStatus("That list doesn't exist! Create a new list to get started.");
+			}
+			// Clear the existingToken after the submission
+			setExistingToken('');
```

Comment:
I would consider removing this line. If the list token does get set then the user will get redirected and you won't have to manually clear state. If the list doesn't exist then it may be because the user typo'd and being able to see what their input was could be helpful.